README.md

```diff
@@ -1,27 +1,33 @@
-# The Rust Programming Language
-
-[![Rust Community](https://img.shields.io/badge/Rust_Community%20-Join_us-brightgreen?style=plastic&logo=rust)](https://www.rust-lang.org/community)
+<p align="center">
+  <img alt="The Rust Programming Language: A language empowering everyone to build reliable and efficient software" src="https://www.rust-lang.org/static/images/rust-social-wide.jpg">
```

Comment:
imo this image is way too big, it's not very useful and just takes away space.

Reply:
yeah, the social image from rust-lang.org has unfortunately the right ratio to be very big; I will try to reduce it's size

Reply:
done.